tools/functional-tester/etcd-tester/main.go

```diff
@@ -73,10 +73,16 @@ func main() {
 		v2:             *isV2Only,
 	}
 
+	lsConfig := &leaseStressConfig{
+		numOfLeases:  10,
+		keysPerLease: 10,
+	}
+
 	c := &cluster{
-		agents:        agents,
-		v2Only:        *isV2Only,
-		stressBuilder: newStressBuilder(*stresserType, sConfig),
+		agents:               agents,
+		v2Only:               *isV2Only,
+		stressBuilder:        newStressBuilder(*stresserType, sConfig),
+		leaseStresserBuilder: newLeaseStressBuilder(lsConfig),
```

Comment:
this is bad, there needs to be a uniform stresser interface or the complexity will explode